packages/flame/lib/src/parallax.dart

```diff
@@ -93,13 +95,17 @@ abstract class ParallaxRenderer {
   /// How to fill the screen with the image, always proportionally scaled.
   final LayerFill fill;
 
+  final FilterQuality filterQuality;
+
   ParallaxRenderer({
     ImageRepeat? repeat,
     Alignment? alignment,
     LayerFill? fill,
+    FilterQuality? filterQuality,
```

Comment:
wouldn't it be simpler to say `this.filterQuality = FilterQuality.low`?

Reply:
We should have all of the arguments in the same fashion I think, I have no idea why the rest of the arguments don't have a default instead of accepting null but now it would be a breaking change to change them.